build.gradle

```diff
@@ -134,7 +139,7 @@ def ensureBuildPrerequisites(requiredJavaVersion, largeResourcesFolder, buildPre
                 "The ClassLoader obtained from the Java ToolProvider is null. "
                 + "A Java $requiredJavaVersion JDK must be installed. $buildPrerequisitesMessage")
     }
-    if (!file(".git").isDirectory()) {
+    if (!looksLikeWereInAGitRepository()) {
```

Comment:
Suggest changing the message below to something like:

`"This doesn't appear to be a git folder. The GATK Github repository must be cloned using \"git clone\" to run the build. + "$buildPrerequisitesMessage")`